CDCgov/prime-reportstream
# 10359 - Add App Insights Tracking for Public Key Tool

This PR ...

Creates App Insights Tracking for Public Key Tool

Test Steps:
1. Log into RS
2. Select a Federal Organization
3. Navigate to Resources Manage Public Key
4. Upload a .pem file that will cause an error.
5. Upload a .pem file that will not cause an error.
6. View App insights where usg_events = 'Public Key' and see the logs that were created.

## Changes
- Added App Insights Tracking for Public Key Tool
- 
![Screenshot 2023-07-28 at 5 15 55 PM](https://github.com/CDCgov/prime-reportstream/assets/102491809/34f89c22-65d2-45fe-ae82-8d871206186d)
![Screenshot 2023-07-28 at 5 16 09 PM](https://github.com/CDCgov/prime-reportstream/assets/102491809/7cea56b5-b0db-4c5d-9361-77507ff171cf)


## Checklist

### Testing

## Linked Issues
- Fixes #10359
